src/bpm/bpm_extractor.py

```diff
@@ -1,8 +1,8 @@
 import sys
 import os
 from multiprocessing import Pool
```

Comment:
pep8 skal den her ned til de andre, tabulate og essentia iirc

Reply:
Multiprocessing is a default package in python, so it shouldn't be moved down.

Reply:
I mean tabulate should go up to multiprocessing